### ISSUE TITLE
About the notion of "static service dispatching"

### ISSUE THREAD

**Issue by Deguello1** [boucadair/can-framework#12]
> usually provide relatively static service dispatching

I don't get what is meant here. Does this mean "service design remains relatively static in a sense that the resources that participate in the delivery (and the operation) of a service remain the same over time, regardless of various, yet possibly affecting criteria like the number of end-users, their location and whether they are in motion or not"? If so, I would rephrase.

**Comment by boucadair** [comment]
@Deguello1, you got the intent.  An edit proposal can be seen at: https://github.com/boucadair/can-framework/commit/2101fe4e5ee022775f7ca2209ee5fba96417d5ac
